app/controllers/invoices_controller.rb

```diff
@@ -1,15 +1,18 @@
 class InvoicesController < ApplicationController
+    
     def index
         @invoices = Invoice.all
     end
     def new
-        
         @invoice = Invoice.new
     end
     def create
         @invoice = Invoice.new(invoice_params)
-       
+        puts invoice_params
+        @invoice.status = FALSE
```

Comment:
Convention: boolean's are just `true` and `false` in lowercase in Ruby (capitals works too but it's not conventional).